skills/README.md

```diff
@@ -0,0 +1,132 @@
+# Skills
+
+## Skills Object
+
+```
+{
+    "name": <name of skill>,
+    "on": <when skill was added>,
+    "by": <who added it>,
+    "for": <reason for adding it> 
+}
+```
+
+## **Requests**
+
+|                       Route                       |         Description          |
+| :-----------------------------------------------: | :--------------------------: |
+|            [GET /skills](#get-skills)             | Returns skills in collection |
+|  [POST /skills/:username](#post-skillsusername)   |  Adds skills to collection   |
+|   [GET /skills/:username](#get-skillsusername)    |   Get skills of given user   |
+| [GET /skills/users/:skill](#get-skillsusersskill) |  Get users with given skill  |

```

Comment:
This should be moved to `users` as it belongs to `users` and the route can be `/users/:skill`